Gemfile

```diff
@@ -9,12 +9,12 @@ gem 'solidus', github: 'solidusio/solidus', branch: branch
 # Needed to help Bundler figure out how to resolve dependencies,
 # otherwise it takes forever to resolve them.
 # See https://github.com/bundler/bundler/issues/6677
-gem 'rails', '>0.a'
+gem 'rails', ENV.fetch('RAILS_VERSION', '>0.a')
 
 # Provides basic authentication functionality for testing parts of your engine
 gem 'solidus_auth_devise'
 
-case ENV['DB']
+case ENV.fetch('DB', nil)
```

Comment:
This was auto-corrected by [Style/FetchEnvVar](https://docs.rubocop.org/rubocop/cops_style.html#stylefetchenvvar).